src/extension.ts

```diff
@@ -104,9 +105,35 @@ export async function activate(context: vscode.ExtensionContext) {
             open(hoverFile);
         }
     });
-
     context.subscriptions.push(clickInclude);
 
+    // Command for getting page ast
+    const getPageAST: vscode.Disposable = vscode.commands.registerCommand('snooty.getPageAST', async () => {
+        const textDocument: vscode.TextDocument = vscode.window.activeTextEditor.document;
+        const fileName: string = textDocument.fileName;
+        
+        // Only valid on .txt site pages
+        if (!fileName.endsWith(".txt")) {
```

Comment:
We need to lift this restriction in the not-too-far-future

Reply:
Yup, this restriction should be removed after [DOCSP-7187](https://jira.mongodb.org/browse/DOCSP-7187) is done